app/src/androidTest/java/com/zulip/android/activities/RecyclerViewTests.java

```diff
@@ -9,7 +9,7 @@
 
 import com.zulip.android.R;
 import com.zulip.android.ZulipApp;
-import com.zulip.android.filters.NarrowFilterToday;
+import com.zulip.android.filters.NarrowFilterByDate;
```

Comment:
@kunall17 These changes are not done by me. These were done in [ec1afe95](https://github.com/zulip/zulip-android/commit/ec1afe95cef354176e851bcb68c00c4eae9ca984). How can I remove them?